packages/inventory-compliance/src/ComplianceRemediationButton/ComplianceRemediationButton.js

```diff
@@ -42,8 +42,8 @@ class ComplianceRemediationButton extends React.Component {
     dataProvider = () => {
         const { allSystems, selectedRules } = this.props;
         const result = { systems: [], issues: [] };
-        allSystems.forEach(async (system) => {
-            system.supported === true && result.systems.push(system.id);
+        allSystems.filter((system) => (system.supported)).forEach(async (system) => {
```

Comment:
Observation: Previously there was a strict comparison to `true`.

Reply:
Yeah?